### DISCUSSION TITLE
copying accompanying json file for each image from Google Takeout to the output folder

### DISCUSSION THREAD

**Discussion by Abdul-Hameed-E** [TheLastGimbus/GooglePhotosTakeoutHelper#220] [General]
I tried the application for 160GB worth of photos/videos from Google Takeout and it worked without any issues. I chose json option in the initial config and got only one JSON for all the photos with the album info inside it. Thank you so much for this application.

Is it possible to copy the original sidecar files from Takeout to the output folder?

I'm trying to read/convert them to xmp sidecar so that face tags and such can accompany the photos in the side car file and easier to import the face tags to other photo management software(trying for digiKam right now). Any tips regarding this is appreciated.

**Comment by TheLastGimbus** [comment]
You mean is it possible to copy their corresponding jsons/contents? Umm, i don't think this will ever be supported, since guessing the photo itself is 1st priority and it's already super complicated ://

But anyway, do jsons contain face data? Show me that please
